requirements/edx/base.txt

```diff
@@ -39,10 +39,6 @@ elasticsearch==0.4.5
 facebook-sdk==0.4.0
 feedparser==5.1.3
 firebase-token-generator==1.3.2
-# Master pyfs has a bug working with VPC auth. This is a fix. We should switch
-# back to master when and if this fix is merged back.
-# fs==0.4.0
```

Comment:
You lost this line in the move. 

It's actually important. When the fix is merged, we don't want to accidentally bump versions of pyfs to latest without first testing. There are major changes, a few of which are a little bit subtle (e.g. unicode handling). We'd want to do at least a sanity check everything continues to work.